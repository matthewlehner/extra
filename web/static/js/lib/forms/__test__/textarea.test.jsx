// @flow

import React from "react";
import renderer from "react-test-renderer";

import Textarea from "../textarea";

const name = "a-textarea";
const onChange = jest.fn();
const value = "content";

describe("Textarea component", () => {
  it("renders properly", () => {
    const tree = renderer.create(
      <Textarea name={name} onChange={onChange} value={value} />
    );

    expect(tree).toMatchSnapshot();
  });
});
