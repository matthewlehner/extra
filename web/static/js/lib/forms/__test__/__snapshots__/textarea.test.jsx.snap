// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Textarea component renders properly 1`] = `
<textarea
  className="form__control"
  id="a-textarea"
  name="a-textarea"
  onChange={[Function]}
  value="content"
/>
`;
